book/website/reproducible-research/open/open-access.md

```diff
@@ -4,32 +4,73 @@
+Likely due to the inefficiency of traditonal journals, preprint servers for other disciplines have grown hugely in popularity over the last 5 years, with the emergence of bioRxiv, medRxiv, ChemRxiv, SocArXiv and many more.
+Preprints allow researchers to get their results out quickly and gives the opportunity to get feedback on a manuscript before submitting it to a journal.
+Some journals will not allow a paper to be submitted if its already been posted on a preprint server as they consider it 'prior publication', however the ongoing popularity of preprints has forced many publishers to embrace preprints, or in the case of journals like eLife, actively encourage them {cite}`elife2020preprints`.
+You can find out more about preprints at [ASAPbio](https://asapbio.org/preprint-info).
+
+(rr-open-access-journal-publishing)=
+### Open Access Journal publishing
+
+Traditionally many journals are subscription journals, meaning their content is behind a paywall. 
+Universities may have subscriptions to that journal, allowing you to access its content if you are based at a subscribing institution or you may be able to pay a one off fee to access an individual article.
+
+The Open Access movement led to the creation of Open Access journals, where the content is open for anyone to access. 
+The number of these journals is still increasing rapidly, with 17,000 listed in the Directory of Open Access Journals [DOAJ](http://www.doaj.org) (data fetched November 2021). 
+
+(rr-open-access-gold-publishing)=
+#### Gold Open Access publishing
+
+Gold Open Access Publishing (or sometimes referred to as libre) can be defined as paying an Article Processing Charge (APC) to a journal so they publish the final version of your article, which is then permanently and freely available online for anyone.
+The author will retain copyright of their article, usually via a Creative Commons licence of their choice, which dictates what others can do with the article. 
+You can find out more about APCs in this [video](https://www.youtube.com/watch?v=QAeTKUpZq_Y).
+
+There can often be misconceptions around publishing in Open Access journals- as they are newer to the system they are often wrongly thought of as 'less rigorous' or 'lower quality' than the 'traditional' journals. 
+Some researchers have also been caught up in what is referred to as 'predatory publishing'- where they are tricked into paying an article processing charge to journals that are fradulent and do not provide editorial processes that check articles for quality and legitimacy.
+Following the [Think.Check.Submit guidelines](https://thinkchecksubmit.org/) can help researchers to avoid predatory journals.
 
-(rr-open-access-whatis-repositories)=
-### Repositories and Self-Archiving
+A criticism around gold Open Access publishing is also the cost. 
+Article Processing Charges (APCs) can generally be around $2000 or in some cases more, which can therefore be prohibitive for authors across the globe. 
```

Comment:
We've already defined APC above so you can just use APC from now on :)

Reply:
done :)